docs/program/apis/cloud.md

```diff
@@ -188,9 +188,281 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The location corresponding to the provided ID.
+
+```python {class="line-numbers linkable-line-numbers"}
+location = await cloud.get_location(location_id="123ab12345")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### UpdateLocation
+
+Change the name of a {{< glossary_tooltip term_id="location" text="parent location" >}} and/or assign it a new location.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `location_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to update.
+- `name` [Optional(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional new name to update the location name to.
+If nothing is passed, the name is not changed.
+- `parent_location_id` [Optional(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional ID of the new location to move the location to.
+If nothing is passed, the location does not move.
+If an empty string is passed, the location moves up to the top level.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid location ID is passed, or if one isn't passed and no location ID was provided at `AppClient` instantiation.
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The newly updated location.
+
+```python {class="line-numbers linkable-line-numbers"}
+# The following line takes the location with ID "abc12abcde" and moves it
+# to be a sub-location of the location with ID "xyz34xxxxx"
+my_updated_location = await cloud.update_location(location_id="abc12abcde", name="", parent_location_id="xyz34xxxxx")
+
+# The following line changes the name of the location without changing its parent location
+my_updated_location = await cloud.update_location(location_id="abc12abcde", name="Land Before Robots")
+
+# The following line moves the location back up to be a top level location without changing its name
+my_updated_location = await cloud.update_location(location_id="abc12abcde", name="", parent_location_id="")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.update_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### DeleteLocation
+
+Delete a {{< glossary_tooltip term_id="location" text="location" >}}.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `location_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to delete.
```

Comment:
```suggestion
- `location_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location.
```
(opt)

Reply:
Possibly superfluous but leaving as-is just to make sure they get it and don't somehow think it's the parent location or some such, and because this is a very short description anyway so I'm not too worried about verbosity here.